manager_dashboard/manager_dashboard/js/usergroup-management.js

```diff
@@ -0,0 +1,115 @@
+$("#add-usergroup-modal").on('show.bs.modal', function() {
+  $('#usergroup-name').val('');
+  $('#usergroup-description').val('');
+})
+
+function submitNewUserGroup() {
+  var nameInput = $('#usergroup-name');
+  var name = (nameInput.val() || '').trim();
+  var descriptionInput = $('#usergroup-description');
+  var description = (descriptionInput.val() || '').trim();
+
+  if (name && name.length <= 50) {
+    $('#add-usergroup-modal-loading-message-container').removeClass('hidden');
+    nameInput.attr('disabled', true);
+    descriptionInput.attr('disabled', true);
+
+    firebase.database().ref('v2/userGroups/' + name).once("value", function(snapshot) {
+      if (snapshot.exists()) {
+        $('#add-usergroup-modal-loading-message-container').addClass('hidden');
+        nameInput.attr('disabled', false);
+        descriptionInput.attr('disabled', false);
+        alert('A group with this name already exists');
+      } else {
+        var updates = {};
+        updates['/v2/userGroups/' + name] = {
+          userGroupName: name,
```

Comment:
Do we need to duplicate the `name` here? Also, can the `userGroupName` key be shortened to `name`, for instance? I'm thinking of data volume and transfer cost in firebase :moneybag:

Reply:
Done!